### DISCUSSION TITLE
advanced/customizing

### DISCUSSION THREAD

**Discussion by giscus** [Mereithhh/vanblog-comment#20] [Announcements]
# advanced/customizing

VanBlog 通过定制化功能允许你在页面中嵌入自己的 HTML、CSS、JS 片段。 设置方法 使用 站点管理/系统设置/定制化 选项卡。 定制化 定制化功能默认开启，你可以在布局设置中关闭它。这样即便设置了定制化相关的代码，也不会生效。 定制化使用 VSCode 同款的代码编辑器，支持自动补全。 自定义 HTML 自定义的 HTML 代码将被插入到...

https://vanblog.mereith.com/advanced/customizing.html

**Comment by Rowlet-Pan** [comment]
怎么样可以让定制化的css只应用于某个页面比如首页呢？
